From 8436f5c6c02391a1547dc60dbcf2ad7651d5b059 Mon Sep 17 00:00:00 2001
From: Michal Marek <mmarek@suse.com>
Date: Tue, 24 Nov 2015 11:01:31 +0100
Subject: [PATCH] kabi: Fix spurious kabi change in mm/util.c
Patch-mainline: Never, kabi

The added include allows genksyms to expand the types further:

../mm/util.c:313: warning: __get_user_pages_fast: modversion changed because of changes in struct mempolicy (became defined)
../mm/util.c:352: warning: get_user_pages_fast: modversion changed because of changes in struct mempolicy (became defined)
../mm/util.c:385: warning: vm_mmap: modversion changed because of changes in struct mempolicy (became defined)

Signed-off-by: Michal Marek <mmarek@suse.com>
---
 mm/util.c | 2 ++
 1 file changed, 2 insertions(+)

diff --git a/mm/util.c b/mm/util.c
index 90b68634420d..909866a6312c 100644
--- a/mm/util.c
+++ b/mm/util.c
@@ -9,7 +9,9 @@
 #include <linux/swapops.h>
 #include <linux/vmalloc.h>
 #include <linux/mman.h>
+#ifndef __GENKSYMS__
 #include <linux/hugetlb.h>
+#endif
 
 #include <asm/uaccess.h>
 
-- 
2.1.4

